composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php": "^7.0",
         "composer/installers": "^1.0",
-        "shopware/shopware": "^5.5",
```

Comment:
just update the version to `^5.6`
I think we should stay with the range operator